mlflow/tracking/fluent.py

```diff
@@ -212,6 +212,40 @@ def log_metric(key, value):
     MlflowClient().log_metric(run_id, key, value, int(time.time()))
 
 
+def log_metrics(metrics):
+    """
+    Log multiple metrics for the current run, starting a run if no runs are active.
+    :param metrics: Dictionary of metric_name: String -> value: Float
+    :returns: None
+    """
+    run_id = _get_or_start_run().info.run_uuid
+    timestamp = int(time.time())
+    metrics_arr = [Metric(key, value, timestamp) for key, value in metrics.items()]
```

Comment:
Given what we know about the possibility of multiple metrics values logged for the same metric name, does it make sense to issue some sort of warning message (not error) right here in the fluent API, so the user knows.